hoomd/md/pytest/test_meshbond.py

```diff
@@ -0,0 +1,195 @@
+import hoomd
+import pytest
+import numpy as np
+
+_harmonic_args = {'k': [30.0, 25.0, 20.0], 'r0': [1.6, 1.7, 1.8]}
+_harmonic_arg_list = [(hoomd.md.mesh.bond.Harmonic,
+                       dict(zip(_harmonic_args, val)))
+                      for val in zip(*_harmonic_args.values())]
+
+_FENE_args = {
+    'k': [30.0, 25.0, 20.0],
+    'r0': [1.6, 1.7, 1.8],
+    'epsilon': [0.9, 1.0, 1.1],
+    'sigma': [1.1, 1.0, 0.9]
+}
+_FENE_arg_list = [(hoomd.md.mesh.bond.FENE, dict(zip(_FENE_args, val)))
+                  for val in zip(*_FENE_args.values())]
+
+_Tether_args = {
+    'k_b': [5.0, 6.0, 7.0],
+    'l_min': [0.7, 0.8, 0.9],
+    'l_c1': [0.9, 1.05, 1.1],
+    'l_c0': [1.1, 1.1, 1.3],
+    'l_max': [1.3, 1.3, 1.5]
+}
+_Tether_arg_list = [(hoomd.md.mesh.bond.Tether, dict(zip(_Tether_args, val)))
+                    for val in zip(*_Tether_args.values())]
+
+
+def get_mesh_bond_and_args():
+    return _harmonic_arg_list + _FENE_arg_list + _Tether_arg_list
+
+
+def get_mesh_bond_args_forces_and_energies():
+    harmonic_forces1 = [[-28.395, 16.393861, 0], [-27.4125, 15.826614, 0],
+                        [-24.93, 14.393342, 0]]
+    harmonic_forces2 = [[0, -32.787722, 0], [0, -31.653229, 0],
+                        [0, -28.786684, 0]]
+    harmonic_forces3 = [[28.395, 16.393861, 0], [27.4125, 15.826614, 0],
+                        [24.93, 14.393342, 0]]
+    harmonic_energies = [17.9172, 20.0385, 20.7168]
+    FENE_forces1 = [[-165.834803, 95.744768, 0], [-9.719869, 5.611769, 0],
+                    [33.483261, -19.331569, 0]]
+    FENE_forces2 = [[0, -191.489537, 0], [0., -11.223537, 0], [0, 38.663139, 0]]
+    FENE_forces3 = [[165.834803, 95.744768, 0], [9.719869, 5.611769, 0],
+                    [-33.483261, -19.331569, 0]]
+    FENE_energies = [82.0225, 48.6153, 33.4625]
+    Tether_forces1 = [[0, 0, 0], [-0.036666, 0.021169, 0],
+                      [-5.358389, 3.093667, 0]]
+    Tether_forces2 = [[0, 0, 0], [0, -0.042339, 0], [0, -6.187334, 0]]
+    Tether_forces3 = [[0, 0, 0], [0.036666, 0.021169, 0],
+                      [5.358389, 3.093667, 0]]
+    Tether_energies = [0, 0.000463152, 0.1472802]
+
+    harmonic_args_and_vals = []
+    FENE_args_and_vals = []
+    Tether_args_and_vals = []
+    for i in range(3):
+        harmonic_args_and_vals.append(
+            (_harmonic_arg_list[i][0], _harmonic_arg_list[i][1],
```

Comment:
```suggestion
            (*_harmonic_arg_list[i],
```
And so on for the other two.